test/runBTagAnalyzer_cfg.py

```diff
@@ -790,12 +822,7 @@
     process.es_prefer_jec = cms.ESPrefer("PoolDBESSource",'jec')
 
 ### to activate the new JP calibration: using the data base
-trkProbaCalibTag = "JPcalib_MC81X_v0"
-if options.runOnData:
-  trkProbaCalibTag = "JPcalib_Data80X_2016_v3"
-if options.JPCalibration:
-	trkProbaCalibTag = options.JPCalibration
-# process.GlobalTag.snapshotTime = cms.string("9999-12-31 23:59:59.000")
+trkProbaCalibTag = options.JPCalibration
```

Comment:
with this change JPCalibration becomes a compulsory option (which already kind of was), in this case you should also set the default to ```'FIXME'``` to ensure a good one is also provided.

Reply:
Done